src/crimson/os/seastore/omap_manager/btree/omap_btree_node_impl.cc

```diff
@@ -747,7 +748,8 @@ omap_load_extent(omap_context_t oc, laddr_t laddr, depth_t depth)
     });
   } else {
     return oc.tm.read_extent<OMapLeafNode>(
-        oc.t, laddr, OMAP_LEAF_BLOCK_SIZE
+        oc.t, laddr,
+	crimson::os::seastore::omap_manager::BtreeOMapManager::get_leaf_size(oc.type)
```

Comment:
nit, no need to be this long, it should be already in the namespace `crimson::os::seastore::omap_manager`.